packages/core/config/storybook/main.js

```diff
@@ -1,11 +1,16 @@
 const fs = require('fs')
 const path = require('path')
 const { merge } = require('webpack-merge')
-const { getPaths } = require('@redwoodjs/internal')
+const { getConfig, getPaths } = require('@redwoodjs/internal')
 const { getSharedPlugins } = require('../webpack.common')
 
+const config = getConfig()
+
 const baseConfig = {
   stories: ['../../../../web/src/**/*.stories.{tsx,jsx,js}'],
+  // spreading false (like ...false) into an object -> nothing (i.e. {});
+  // great for when we don't wan't to include the property.
+  ...(config.web.a11y && { addons: ['@storybook/addon-a11y'] }),
```

Comment:
Here's a nice way to clean this up:


```suggestion
  addons: [
     config.web.a11y && '@storybook/addon-a11y'
  ].filter(Boolean)
```

This pattern is basically bread 'n butter of configurations